packages/main/test/specs/DateRangePicker.spec.js

```diff
@@ -0,0 +1,46 @@
+const assert = require("chai").assert;
```

Comment:
Please add tests for firstDateValue, lastDateValue - as public getters it is good to have them covered.